src/Driver/GeckoDriver/Downloader.php

```diff
@@ -111,50 +109,34 @@ private function downloadArchive(Driver $driver): string
         return $temporaryFile;
     }
 
-    /**
-     * @throws NotImplemented
-     */
+    /** @throws NotImplemented */
     private function getDownloadPath(Driver $driver): string
     {
         return self::DOWNLOAD_BASE_PATH . sprintf(
             'v%s/geckodriver-v%s-%s%s',
-            $driver->version()->toString(),
-            $driver->version()->toString(),
+            $driver->version->toString(),
+            $driver->version->toString(),
             $this->getOsForDownloadPath($driver),
-            $this->getArchiveExtension($driver)
+            $this->getArchiveExtension($driver),
         );
     }
 
-    /**
-     * @throws NotImplemented
-     */
+    /** @throws NotImplemented */
```

Comment:
The `@throws` docs might need to be checked through the repo. As the switching to match statements caused quite a few exceptions to not be thrown anymore. (This is one of the cases)